### PR TITLE
RV-1835 4. Increase the font of author of quote

### PR DESCRIPTION
**Pull request by dave-rv**

### CODE REVIEW

**Comment by dave-rv** [comment]
Not sure why the // comments are being added/removed?

**Comment by middric** [comment]
Your editor is removing trailing spaces thats all :)

File: sass/_components.scss [hunk 1]
```diff
@@ -43,7 +43,7 @@
 
     cite {
         font-family: 'Function Pro', sans-serif;
-        font-size: 0.75rem;
+        font-size: 1rem;
```
Comment (middric) [inline sass/_components.scss]: `font-size: $size-6;`

**Comment by dave-rv** [comment]
Thanks, disabled the whitespace setting!

**Pull request merged**